cosmos-gui/resources/mysql_upgrade_0.1.0-0.2.0.sql

```diff
@@ -0,0 +1,8 @@
+USE cosmos;
+
+ALTER TABLE cosmos_user MODIFY registration_time TIMESTAMP DEFAULT "0000-00-00 00:00:00";
+ALTER TABLE cosmos_user ADD COLUMN last_access_time TIMESTAMP DEFAULT "0000-00-00 00:00:00";
+ALTER TABLE cosmos_user ADD COLUMN hdfs_used BIGINT NOT NULL;
+ALTER TABLE cosmos_user ADD COLUMN fs_used BIGINT NOT NULL;
+ALTER TABLE cosmos_user MODIFY hdfs_quota BIGINT NOT NULL;
+UPDATE cosmos_user SET hdfs_quota=1073741824*hdfs_quota;
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/3d4b5ed9058b3b2b8048135f588ef79eab0e8ad3